<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HP" ignored="0" name="Test" tests="9" failures="3" timestamp="12 Apr 2022 05:03:36 GMT" time="0.281" errors="0">
  <testcase name="testA1" time="0.029" classname="org.test.A">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.A.testA1(A.java:13)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testA1 -->
  <testcase name="testA2" time="0.008" classname="org.test.A"/>
  <testcase name="testA3" time="0.003" classname="org.test.A"/>
  <testcase name="testb1" time="0.004" classname="org.test.B"/>
  <testcase name="testb2" time="0.004" classname="org.test.B">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.B.testb2(B.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testb2 -->
  <testcase name="testb3" time="0.003" classname="org.test.B"/>
  <testcase name="Testc1" time="0.005" classname="org.test.C"/>
  <testcase name="Testc2" time="0.005" classname="org.test.C"/>
  <testcase name="Testc3" time="0.004" classname="org.test.C">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.C.Testc3(C.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Testc3 -->
</testsuite> <!-- Test -->
